org.eclipse.nebula.widgets.nattable.examples.e4.rcp.feature/feature.xml

```diff
@@ -548,11 +506,4 @@
          version="0.0.0"
          unpack="false"/>
 
-   <plugin
```

Comment:
I added com.ibm.icu.base in the past to reduce the size of the product, because com.ibm.icu is a huge bundle from which only a very limited set is actually needed. Has this changed in the meanwhile? Is com.ibm.icu not needed anymore?

Reply:
It doesn't exist anymore.  If some bundle needs something, it will be found.  I don't see any direct dependencies on it.  Again, like the above.  The dependencies are being over-specified and in a way that's very restrictive and makes it impossible to use your feature except in very restricted installations.  Hasn't anyone complained about that?  Or maybe no one is using the feature because it's not so useful?

Reply:
Nobody ever used it, because it is not published. It is only part of the NatTable Examples Application. And it is used to keep the product size as small as possible. That was necessary in the past, as the size was almost twice as big as needed. But probably this is not the case anymore and I can drop that feature completely. The main intention back then was to get rid of com.ibm.icu and replace it with com.ibm.icu.base. If this is not needed anymore, I probably can switch to the e4.rcp.feature provided by the Platform. Will check this.